### ISSUE TITLE
Inconsistent keyboard shortcut behaviour on macOS

### ISSUE THREAD

**Issue by krzysztof-callstats** [sedwards2009/extraterm#213]
I'm using Extraterm version `0.44.1` on macOS Mojave version `10.14.6 (18G95)`. Recently I started experiencing some weird problems with keyboard shortcuts. I was unable to open to Command Palette and upon experimenting it turns out that it does matter in which order I press the `cmd+shift+P` sequence. Neither `cmd->shift->P` nor `shitf->cmd->P` works, but both `p->cmd->shift` and `p->shift->cmd` work perfectly fine. How is it even possible?

**Comment by sedwards2009** [comment]
I wasn't able to reproduce this problem on macOS High Sierra 10.13.4.

**Comment by sedwards2009** [comment]
Are you using a keyboard set up for the Polish language? or one which matches English?

**Comment by krzysztof-callstats** [comment]
Then I use Polish Pro version. It is identical with English one, with the addition of couple `option+<someletter>` special letters. In fact, I switched to the English and the problem still exists.

**Comment by krzysztof-callstats** [comment]
I discovered that the same shortcut doesn't work in my vscode. I dug around the web and found that if all "standard" fixes are not working, then it most likely is some other app that intercepts these shortcuts. This was exactly what was happening in my case. Most of the time people mentioned software like Voila, but for me, it was TickTick that was the culprit. Disabled the shortcut in the settings and everything is fine again!

TL;DR; It was not Extraterm's fault!

**Comment by sedwards2009** [comment]
ok, good.
